library/global.ml

```diff
@@ -104,6 +105,13 @@ let add_modtype id me inl = globalize (Safe_typing.add_modtype (i2l id) me inl)
 let add_module id me inl = globalize (Safe_typing.add_module (i2l id) me inl)
 let add_include me ismod inl = globalize (Safe_typing.add_include me ismod inl)
 
+let open_section ~poly = globalize0 (Safe_typing.open_section ~poly)
+let close_section fs =
+  (* TODO: use globalize0_with_summary *)
```

Comment:
Should this be addressed now? If not, maybe write what the difficulty is so that we remember what is to be done exactly.

Reply:
No, it's not possible right now due to the inversion of control between summaries and section closing in `Declare`. There is a loop to untie when discharging of constants happens kernel-side.